index.html

```diff
@@ -2541,7 +2581,7 @@ <h3>The <dfn>NFCScanOptions</dfn> dictionary</h3>
       <pre class="idl">
         dictionary NFCScanOptions {
           USVString id = "";
-          NDEFRecordType recordType;
+          USVString recordType;
           USVString mediaType = "";
```

Comment:
HTML spec uses DOMString for media type... guess we should do the same?

Things that include urls should be USVString I have been told

Reply:
In Absolute URL the recordType would contain a URL

Reply:
I thought we just set it to "absolute-url" and put the url in payload?

Reply:
actually we should do that, we set it to "url" now

![image](https://user-images.githubusercontent.com/1101339/67011142-4b23d400-f0ef-11e9-9a1c-ed62343f0b9b.png)

Reply:
IMHO we should not deviate from NDEF semantics here. For TNF=3 let's use the TYPE field for the URL.

Reply:
Maybe we can _also_ set `data` to the same, but `recordType` should contain the URL as well.

Reply:
These are not supposed to contain payload so it's fine

Reply:
Maybe we should expose TNF?